apps.json

```diff
@@ -3557,7 +3557,6 @@
     "proxitok": {
         "category": "social_media",
         "level": 7,
-        "state": "notworking",
```

Comment:
also uuuh why change this

Reply:
Again the script did this. but studying file apps.json, there is both status on the app : 
```
   "proxitok": {
        "category": "social_media",
        "level": 7,
        "state": "notworking",
        "potential_alternative_to": [
            "TikTok"
        ],
        "state": "working",
        "url": "https://github.com/YunoHost-Apps/proxitok_ynh"
    },
```

Reply:
uuugh indeed x_x